js/ui/overlay.js

```diff
@@ -1,2 +1,7 @@
 import Overlay from './overlay/ui.overlay';
-export default Overlay;
+
+const baseZIndex = Overlay.baseZIndex;
+export {
+    Overlay,
```

Comment:
Do we need this export? Overlay is private and all internal imports uses the `overlay/ui.overlay' module now. So who is it for?

Reply:
left it just for more consistence with other editors. I agree, removed it